src/extensionFunctions.ts

```diff
@@ -155,8 +155,10 @@ export async function startTerminal(...rest: any[]): Promise<void>{ //name:strin
 		}
 	}
 	if (name) {
-		if (findTerminal(name)) {
-			throw new Error(`Terminal ${name} already exists`);
+		const oldTerm = findTerminal(name);
+		if (oldTerm) {
+			oldTerm.show();
+			return;
```

Comment:
the comment of the method is no more accurate. There is not an handle to the terminal in this case https://issues.redhat.com/browse/FUSETOOLS2-1008